test/e2e/placement_test.go

```diff
@@ -34,14 +38,43 @@ var _ = Describe("placing resources using a CRP with no placement policy specifi
 			},
 			Spec: placementv1beta1.ClusterResourcePlacementSpec{
 				ResourceSelectors: workResourceSelector,
+				Strategy: placementv1beta1.RolloutStrategy{
+					Type: placementv1beta1.RollingUpdateRolloutStrategyType,
+					RollingUpdate: &placementv1beta1.RollingUpdateConfig{
+						UnavailablePeriodSeconds: pointer.Int(2),
+					},
+				},
 			},
 		}
 		Expect(hubClient.Create(ctx, crp)).To(Succeed(), "Failed to create CRP")
 	})
 
 	It("should place the resources on all member clusters", func() {
 		for idx := range allMemberClusters {
-			workResourcesPlacedActual := workNamespaceAndDeploymentPlacedOnClusterActual(allMemberClusters[idx])
+			workResourcesPlacedActual := workNamespaceAndConfigMapPlacedOnClusterActual(allMemberClusters[idx])
+			Eventually(workResourcesPlacedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to place work resources on member cluster")
+		}
+	})
+
+	It("should update CRP status as expected", func() {
+		crpStatusUpdatedActual := crpStatusUpdatedActual()
+		Eventually(crpStatusUpdatedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to update CRP status as expected")
+	})
+
+	It("can update the resource", func() {
+		// Get the config map.
+		configMap := &corev1.ConfigMap{}
+		Expect(hubClient.Get(ctx, types.NamespacedName{Namespace: workNamespaceName, Name: appConfigMapName}, configMap)).To(Succeed(), "Failed to get config map")
+
+		configMap.Data = map[string]string{
+			"data": "updated",
+		}
+		Expect(hubClient.Update(ctx, configMap)).To(Succeed(), "Failed to update config map")
+	})
+
+	It("should place the resources on all member clusters", func() {
+		for idx := range allMemberClusters {
+			workResourcesPlacedActual := workNamespaceAndConfigMapPlacedOnClusterActual(allMemberClusters[idx])
```

Comment:
does this check if the config data is changed?

Reply:
Hi Ryan! Yeah, this one retrieves the data from the hub cluster as the baseline, so it already includes the change.